addons/wowl/static/tests/webclient/web_client_integrated_tests.ts

```diff
@@ -1900,26 +1901,32 @@ QUnit.module("Action Manager Legacy Tests Porting", (hooks) => {
     baseConfig.actionRegistry!.remove("ClientAction");
   });
 
-  QUnit.skip("load a window action without id (in a multi-record view)", async function (assert) {
-    // unskip: need to store the action in session storage and implement the restore
-    assert.expect(14);
+  QUnit.test("load a window action without id (in a multi-record view)", async function (assert) {
+    assert.expect(15);
 
-    // var RamStorageService = AbstractStorageService.extend({
-    //   storage: new RamStorage(),
-    // });
-    // services: {
-    //   session_storage: RamStorageService,
-    // },
-    // testUtils.mock.intercept(
-    //   actionManager,
-    //   "call_service",
-    //   function (ev) {
-    //     if (ev.data.service === "session_storage") {
-    //       assert.step(ev.data.method);
-    //     }
-    //   },
-    //   true
-    // );
+    const localStorage = baseConfig.browser!.localStorage;
+    baseConfig.browser!.localStorage = Object.assign(Object.create(localStorage!), {
+      getItem(k: any) {
+        assert.step(`getItem local ${k}`);
+        return localStorage!.getItem(k);
+      },
+      setItem(k: any, v: any) {
+        assert.step(`setItem local ${k}`);
+        return localStorage!.setItem(k, v);
+      },
+    });
```

Comment:
do you need the local storage here? I think that we don't care about what the list does in this test